metadata-ingestion/src/datahub/ingestion/extractor/mce_extractor.py

```diff
@@ -13,6 +13,13 @@
 )
 from datahub.metadata.schema_classes import UsageAggregationClass
 
+try:
+    from black import FileMode, format_str
+except ImportError:
+    has_black = False
+else:
+    has_black = True
+
```

Comment:
Instead of the extra indicator variable, could we do this? It'd also be more similar to what other Python packages do.

```python
try:
  import black
except ImportError:
  black = None
```